### PR TITLE
add rcServo,rcIMU,rcBMP functions

### PR DESCRIPTION
**Pull request by vaishnav98**
changes made to node-roboticscape: https://github.com/jadonk/node-roboticscape/pull/5

tested on:
BeagleBone Blue BeagleBoard.org Debian Image 2018-03-05
Linux beaglebone 4.9.105-ti-r113
rc_version : 0.4.4

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ var autorun = require('./autorun');
 var server = require('./server');
 var socketHandlers = require('./socket_handlers');
 var ffi = require('./ffiimp');
-//var rc = require('./rc');
+var rc = require('./rc');
```
Comment (jadonk) [inline src/index.js]: Requires an package.json update to 0.0.8.

Comment (jadonk) [reply src/index.js]: Also, can we use the optional require here in case the package doesn't install nice?

File: src/index.js [hunk 2]
```diff
@@ -773,9 +773,9 @@ for (var x in socketHandlers) {
 for (var x in ffi) {
     f[x] = ffi[x];
 }
-/*for(var x in rc) {
-    exports[x] = rc[x];
-}*/
+for (var x in rc) {
```
Comment (jadonk) [inline src/index.js]: Can you handle the condition that the library isn't installed nice?

**Comment by vaishnav98** [comment]
@jadonk i have updated the version in package.json ,"roboticscape" is already listed as an optional dependency, so if there is any use of obsolete functions(such as when the installed version of the library is old), shouldn't the optional dependency build fail without affecting other things, also the optional require is used inside rc.js : https://github.com/jadonk/bonescript/blob/0c7c11f0fd4041cee5296c4c11f7351f192e2ae7/src/rc.js#L3 , so would these not be necessary to deal with the fail cases you discussed,please let me know if i missed anything

**Pull request merged**